packages/components/src/toolbar/toolbar-group/toolbar-group-collapsed.tsx

```diff
@@ -28,6 +28,7 @@ function ToolbarGroupCollapsed( {
 			controls={ controls }
 			toggleProps={ {
 				...internalToggleProps,
+				size: 'compact',
```

Comment:
As seen in the default Storybook story:

<img width="515" alt="ToolbarGroupCollapsed" src="https://github.com/user-attachments/assets/96dff0df-bcca-4d83-bf89-3583dc2c02e2" />